.trivyignore

```diff
@@ -11,6 +11,9 @@ CVE-2023-25173
 CVE-2022-3821
 CVE-2022-41723
 
-# ubuntu base vulnerabilities
-CVE-2022-48303
-CVE-2022-3821
+CVE-2022-41723
```

Comment:
add some description & severity about these vulns. so we can use that info to assess and handle them.

Reply:
Updated